spec/jobs/project_storage_provider_initialization_job_spec.rb

```diff
@@ -5,35 +5,35 @@
   let!(:storage_provider) { FactoryGirl.create(:storage_provider, :swift) }
   let(:prefix) { Rails.application.config.active_job.queue_name_prefix }
   let(:prefix_delimiter) { Rails.application.config.active_job.queue_name_delimiter }
-
+  let(:job_transaction) { described_class.initialize_job(project) }
 
   it { is_expected.to be_an ApplicationJob }
   it { expect(prefix).not_to be_nil }
   it { expect(prefix_delimiter).not_to be_nil }
   it { expect(described_class.queue_name).to eq("#{prefix}#{prefix_delimiter}project_storage_provider_initialization") }
 
-  context 'perform_now', :vcr do
-    it 'should require a project argument' do
-      expect {
-        described_class.perform_now
-      }.to raise_error(ArgumentError)
-      expect {
-        described_class.perform_now(storage_provider: storage_provider)
-      }.to raise_error(ArgumentError)
-    end
-
-    it 'should require a storage_provider argument' do
-      expect {
-        described_class.perform_now
-      }.to raise_error(ArgumentError)
-      expect {
-        described_class.perform_now(project: project)
-      }.to raise_error(ArgumentError)
-    end
+  it {
```

Comment:
These `expect { ... }.to raise_error` calls should be in their own `it {}` example blocks.